AccessibilityInsightsForAndroidService/.gitignore

```diff
@@ -10,6 +10,8 @@
 /.idea/modules.xml
 /.idea/navEditor.xml
 /.idea/workspace.xml
+.idea/deploymentTargetDropDown.xml
```

Comment:
should this have a `/` before it like the others?

Reply:
You're right
I corrected it in my last commit